test/models/user_test.exs

```diff
@@ -18,6 +19,29 @@ defmodule Handsup.UserTest do
     refute changeset.valid?
   end
 
+  test "owned? without group_id" do
+    user = %User{id: 1}
+    event = %{id: 1}
+    refute User.owned?(user, event)
+  end
+
+  test "owned? with group_id" do
```

Comment:
(maybe) how about rename test name like:
`"owned? with group_id"` to `"owned? with group_id that user not owned"`
`"owned? with own group_id"` to `"owned? with group_id that user owned"`